test/src/test/java/hudson/model/ParametersTest.java

```diff
@@ -270,7 +270,7 @@ static class MyMarkupFormatter extends MarkupFormatter {
         @Override
         public void translate(String markup, @NonNull Writer output) throws IOException {
             Matcher m = Pattern.compile("[<>]").matcher(markup);
-            StringBuffer buf = new StringBuffer();
+            StringBuilder buf = new StringBuilder();
```

Comment:
See [this page](https://errorprone.info/bugpattern/JdkObsolete) for the reasoning.